.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build
+
+on: push
+env:
+  ACTIVATOR_VERSION: '1.3.10'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Install metafacture-core snapshot
+      run: |
+        git clone https://github.com/metafacture/metafacture-core.git
+        cd metafacture-core
+        ./gradlew install
+    - name: Build with Maven
+      run: mvn --batch-mode --update-snapshots verify
+    - name: Build with play
+      run: |
+        cd web
+        wget http://downloads.typesafe.com/typesafe-activator/${{env.ACTIVATOR_VERSION}}/typesafe-activator-${ACTIVATOR_VERSION}-minimal.zip ;unzip typesafe-activator-${{env.ACTIVATOR_VERSION}}-minimal.zip
```

Comment:
I don't understand why one time `${{env.ACTIVATOR_VERSION}}` and one time `${ACTIVATOR_VERSION}` is used.

Reply:
Good catch!
I wonder why it worked nonetheless (https://github.com/hbz/lobid-resources/runs/1707357417?check_suite_focus=true#step:6:3) - however, fixed this with last commit.